Automattic/wp-calypso
# Partner Portal: Update partner portal layout to follow the Pro Dashboard spacing.

Jetpack Cloud has some inconsistency in the layout spacing, making the experience a bit disconnected. This is a step to make the Jetpack Cloud experience follow consistent layout spacing.

Related to 1202619025189113-as-1204969880105860

Note that this task is a follow up task to 1202619025189113-as-1204839784815643

## Proposed Changes

The PR updates the Partner portal-related screens to follow the layout spacing used in the Pro Dashboard. I added official layout component for the Partner portal to reduce boilerplate codes and at the same time enforce consistent spacing. 

**Before**
<img width="1728" alt="Screen Shot 2023-07-06 at 3 25 10 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/dc25807e-4559-490f-a07f-e9876bfc5db1">

**After**
<img width="1728" alt="Screen Shot 2023-07-06 at 3 25 47 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/3cf5fb5a-e8fb-4945-ae88-b7d2d3d715a0">


## Testing Instructions

Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb. Make sure to switch it back to the previous type.

* Use the Jetpack cloud live link below and test the following screens

| Screen  | URL |
| ------------- | ------------- |
| Billing  | /partner-portal/billing  |
| Issue License  | /partner-portal/issue-license  |
| Assign License  | /partner-portal/assign-license  |
| Payment Methods | /partner-portal/payment-methods |
| Invoices | /partner-portal/invoices |
| Prices | /partner-portal/prices |



## Pre-merge Checklist


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1204969880105860
